ansible/roles/test/tasks/add_container_to_inventory.yml

```diff
@@ -0,0 +1,24 @@
+- name: Add/update the public key in the "{{ lookup('env','HOME') + '/.ssh/known_hosts' }}"
+  delegate_to: localhost
+  known_hosts:
+    name: "{{ ansible_host }}"
+    key: "{{ lookup('pipe', 'ssh-keyscan ' + ansible_host) }}"
+    state: "present"
+
+- name: Generate an OpenSSH keypair
+  delegate_to: localhost
+  openssh_keypair:
+    path: "{{ lookup('env','HOME') + '/.ssh/id_rsa' }}"
```

Comment:
>id_rsa [](start = 44, length = 6)

Call it `id_rsa_ansible_temp`, so user may be less surprised.

Reply:
If we changed id_rsa to another name then we should specify `IdentityFile` for a specific host in `~/.ssh/config as there are no other ways to put this name to docker-ce-cli from ansible